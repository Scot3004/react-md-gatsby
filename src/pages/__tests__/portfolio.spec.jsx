import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Portfolio from '../portfolio';

it('renders correctly', () => {
  const tree = shallow(<Portfolio />);
  expect(toJson(tree)).toMatchSnapshot();
});
