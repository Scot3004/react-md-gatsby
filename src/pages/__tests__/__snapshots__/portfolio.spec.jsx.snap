// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="portfolio-container"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Portfolio | Sergio Orozco
    </title>
    <link
      href="https://www.secorto.com/portfolio/"
      rel="canonical"
    />
  </HelmetWrapper>
  <div>
    Hola
  </div>
</div>
`;
